src/core/model_config.proto

```diff
@@ -547,6 +547,37 @@ message ModelOptimizationPolicy
   //@@
   message Cuda
   {
+    //@@    .. cpp:var:: message GraphSpec
+    //@@
+    //@@       Specification of the CUDA graph to be captured.
+    //@@
+    message GraphSpec
+    {
+      //@@      .. cpp:var:: message Dims
+      //@@
+      //@@         Specification of tensor dimension.
+      //@@
+      message Dims {
+        //@@        .. cpp:var:: int64 dim (repeated)
+        //@@
+        //@@           The dimension.
+        //@@
+        repeated int64 dim = 1;
+      }
+
+      //@@      .. cpp:var:: int32 batch_size
+      //@@
+      //@@         The batch size of the CUDA graph.
+      //@@
+      int32 batch_size = 1;
+
+      //@@      .. cpp:var:: map<string, Dims> input
+      //@@
+      //@@         The specification of the inputs.
```

Comment:
This is the shape captured for each input. Does the shape include the batch dimension? Is 'batch_size' handled like max_batch_size? Can you add some more comments to clarify. In general I'm ok with the format.

Reply:
I will update that. The `batch_size` is handled like `max_batch_size` and the shape doesn't include batch dimension to match the convention in model config.